rllib/tuned_examples/regression_tests/cartpole-ppo-torch.yaml

```diff
@@ -0,0 +1,11 @@
+cartpole-ppo-torch:
+    env: CartPole-v0
+    run: PPO
+    stop:
+        episode_reward_mean: 150
+        timesteps_total: 100000
+    config:
+        num_workers: 1
+        batch_mode: complete_episodes
+        observation_filter: MeanStdFilter
+        use_pytorch: true
```

Comment:
@sven1977 it looks like this is failing to reach 150 reward in time:
```
Result logdir: /root/ray_results/cartpole-ppo-torch
Number of trials: 1 (1 TERMINATED)
+--------------------------+------------+-------+--------+------------------+-------------+----------+
| Trial name               | status     | loc   |   iter |   total time (s) |   timesteps |   reward |
|--------------------------+------------+-------+--------+------------------+-------------+----------|
| PPO_CartPole-v0_816b8e8a | TERMINATED |       |     25 |          392.778 |      103299 |   136.79 |
+--------------------------+------------+-------+--------+------------------+-------------+----------+


== Status ==
Memory usage on this node: 16.6/125.8 GiB
Using FIFO scheduling algorithm.
Resources requested: 0/32 CPUs, 0/0 GPUs, 0.0/80.03 GiB heap, 0.0/12.84 GiB objects
Result logdir: /root/ray_results/cartpole-ppo-torch
Number of trials: 1 (1 TERMINATED)
+--------------------------+------------+-------+--------+------------------+-------------+----------+
| Trial name               | status     | loc   |   iter |   total time (s) |   timesteps |   reward |
|--------------------------+------------+-------+--------+------------------+-------------+----------|
| PPO_CartPole-v0_816b8e8a | TERMINATED |       |     25 |          392.778 |      103299 |   136.79 |
+--------------------------+------------+-------+--------+------------------+-------------+----------+


Regression test flaked, retry 2
Regression test FAILED
```

Not sure why this is, perhaps some bad default hyperparam?

Reply:
https://amplab.cs.berkeley.edu/jenkins/job/Ray-PRB/20853/console

Reply:
```
++ docker run --rm --shm-size=20G --memory=20G 4c9b19c18f331dc9266e7869bb344b87256ee6c894f0777e84244c9f389b4720 /ray/ci/suppress_output python /ray/rllib/tests/run_regression_tests.py /ray/rllib/tuned_examples/regression_tests/cartpole-ppo-torch.yaml
WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
This command has been running for more than 5 minutes...
This command has been running for more than 10 minutes...
This command has been running for more than 15 minutes...
This command has been running for more than 20 minutes...
This command has been running for more than 25 minutes...

real	29m24.312s
user	531m52.736s
sys	256m30.880s
```

Looks like it passed but took 30 minutes, so it might be a system-level perf thing. 30 minutes for 100k timesteps does seem too slow though.

Reply:
Ok since everything else is passing, I just deleted this file. We should restore it in another PR.

Reply:
Runs reasonably fast locally. 100k time steps in 2-3 min.

Reply:
"system-level perf thing" I think so. The cloud run on Atari yesterday was reasonably fast. No results yet vs tf, but definitely not as slow as 100k ts per 30min.